lib/billing.rb

```diff
@@ -0,0 +1,24 @@
+class Billing
+
+  def sum(order)
+    total = 0
+    order.each do |x|
+      if x.key?(:name)
+        total += x[:price] * x[:quantity]
+      end
+    end
+    total
+  end
+
+  def check_sum(order, order_total = sum(order))
+    total = 0
+    order.each do |x|
+      if x.key?(:name)
+        total += x[:price] * x[:quantity]
+      end
+    end
+    raise "Sum is not correct!" unless order_total == total
+    true
```

Comment:
Usually if you have to explicitly return true in a method there is a different approach you can take by refactoring.

Focus on the return value of the method and what the method needs to do to get there.